addons/l10n_pe_website_sale/tests/test_l10n_pe_website_sale.py

```diff
@@ -0,0 +1,21 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+
+@odoo.tests.tagged('post_install_l10n', 'post_install', '-at_install')
+class TestWebsiteSalePe(odoo.tests.HttpCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env['product.product'].create({
+            'name': 'Test Product',
+            'standard_price': 70.0,
+            'list_price': 70.0,
+            'website_published': True,
+        })
+        website = cls.env['website'].get_current_website()
+        website.company_id.account_fiscal_country_id = website.company_id.country_id = cls.env.ref('base.pe')
```

Comment:
no need for a setup class if you only have one test.
Also, please add some assertions to make sure the setup is the one you're expecting (i.e. no vat set on the admin partner)